.config/vim/vimrc

```diff
@@ -8,59 +38,33 @@ filetype off
 call plug#begin('~/.vim/plugged')
 Plug 'neoclide/coc.nvim', {'branch': 'release'}
 Plug 'elmcast/elm-vim'
-Plug 'tpope/vim-endwise'
 Plug 'tpope/vim-fugitive'
-Plug 'tpope/vim-haml'
 Plug 'tpope/vim-markdown'
 Plug 'tpope/vim-rails'
 Plug 'tpope/vim-surround'
 Plug 'tpope/vim-repeat'
-Plug 'nathanaelkane/vim-indent-guides'
-Plug 'SirVer/ultisnips'
-Plug 'godlygeek/tabular'
-Plug 'majutsushi/tagbar'
-Plug 'vim-ruby/vim-ruby'
 Plug 'vim-airline/vim-airline'
 Plug 'vim-airline/vim-airline-themes'
-Plug 'HerringtonDarkholme/yats.vim'
 Plug 'vim-scripts/nginx.vim'
-Plug 'vim-scripts/AutoTag'
 Plug 'vim-scripts/haskell.vim'
 Plug 'vim-scripts/LargeFile'
 Plug 'mattn/emmet-vim'
-Plug 'pangloss/vim-javascript'
-Plug 'docunext/closetag.vim'
-Plug 'nono/vim-handlebars'
-Plug 'briancollins/vim-jst'
```

Comment:
if you add polyglot you can remove all the syntax plugins

Reply:
can you link  me?

Reply:
https://github.com/sheerun/vim-polyglot